pkg/kn/commands/service/describe.go

```diff
@@ -99,10 +101,16 @@ func NewServiceDescribeCommand(p *commands.KnParams) *cobra.Command {
 		Example:           describe_example,
 		ValidArgsFunction: commands.ResourceNameCompletionFunc(p),
 		RunE: func(cmd *cobra.Command, args []string) error {
+			var serviceName string
 			if len(args) != 1 {
-				return errors.New("'service describe' requires the service name given as single argument")
+				c, _ := plugin.NewContextManager()
+				serviceName = c.GetDefault()["service"]
+				if serviceName == "" {
+					return errors.New("'service describe' requires the service name given as single argument")
+				}
```

Comment:
@rhuss what would you say about using Golang's build-in Context since `map[string]string` is our context now? We can pass it through `ExecuteWithContext()` function that's already part of cobra.

Or shoud we be a bit more bold here and inject the params directly into args & flags?